dev-vcs/git/git-2.45.2.recipe

```diff
@@ -66,6 +66,14 @@ PROVIDES_gui="
 	cmd:git_gui = $portVersionCompat
 	cmd:gitk = $portVersionCompat
 	"
+PROVIDES_perl="
+	git${secondaryArchSuffix}_perl = $portVersionCompat
+	"
+if [ -n "$secondaryArchSuffix" ]; then
+	PROVIDES_perl+="
+		git_perl = $portVersionCompat
+		"
+fi
```

Comment:
Reason for this block: `git_perl` is effectively a perl module package, similar to the ones in dev-perl. This could then be used as a dependency of another any-arch perl package.